configgen/src/test/java/com/yahoo/config/codegen/DefLineParsingTest.java

```diff
@@ -246,4 +247,24 @@ void testRestartSpecification() {
         assertTrue(r1.getRestart());
     }
 
+    @Test
+    void testParseOptionalPathWithDefault() {
+        DefLine l = new DefLine("pathWithDef path optional");
+
+        assertEquals("pathWithDef", l.getName());
+        assertNull(l.getDefault());
+        assertTrue(l.isOptional());
+        assertEquals("optionalPath", l.getType().getName());
+    }
+
+    @Test
+    void testParsPathWithDefault() {
```

Comment:
There seems to be a mismatch between the test name and the actual def lines. We probably want to test optional path both with and without default?

Reply:
Right, thanks, leftover from when this was implemented differently, rewritten and renamed now